### PR TITLE
[FrameworkBundle] Add autowiring alias for PSR-14

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

**Comment by Tobion** [comment]
I think requiring Psr\EventDispatcher would be developer friendlier. Currently developing against `\Symfony\Contracts\EventDispatcher\EventDispatcherInterface` is quite annoying because there are multiple definitions: one with and one without extending the PSR interface. In phpstorm you cannot simply follow the definition because it asks you to chose:

multiple-eventdispatcher

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php [hunk 1]
```diff
@@ -154,6 +155,10 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('fragment_renderer.xml');
         $loader->load('error_catcher.xml');
 
+        if (interface_exists(PsrEventDispatcherInterface::class)) {
```
Comment (derrabus) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php]: Do we actually need the interface to exist for creating the alias? 🤔

Comment (nicolas-grekas) [reply src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php]: We can but the error will be cryptic and runtime, while the missing alias will lead to a compile time error.

**Review by lyrixx — APPROVED** [review]

**Review by derrabus — APPROVED** [review]

**Review by Tobion — APPROVED** [review]

**Comment by Tobion** [comment]
Thank you @nicolas-grekas.

**Pull request merged**